### ISSUE TITLE
Material frame adding shadow even when HasShadow="False"

### ISSUE THREAD

**Issue by 07vij1987** [roubachof/Sharpnado.Presentation.Forms#50]
**Platform (please complete the following information):**
 - OS: Both
 - Xamarin.Forms: [3.4.0.1029999]

I have used material frame in place of frame so that I can use elevation property.  I have set HasShadow="False". When I rotate to landscape or vice versa, shadow is added automatically.

[page.txt](https://github.com/roubachof/Sharpnado.Presentation.Forms/files/3116628/page.txt)

![image](https://user-images.githubusercontent.com/43951338/56731837-16a31100-6779-11e9-92d5-3df2c873742c.png)

```

**Screenshots (if applicable)**
If applicable, add screenshots to help explain your problem.

**Comment by roubachof** [comment]
Don't use the HasShadow property on MaterialFrame, just put Elevation=0

**Comment by 07vij1987** [comment]
I need elevation. if I try with elevation > 0 without HasShadow property, issue still persists.

**Comment by roubachof** [comment]
Sorry I can't reproduce:

1. If I set the elevation to 0 and I rotate, no shadow is added as wanted
![image](https://user-images.githubusercontent.com/596903/59526630-ab083500-8ed9-11e9-9ec2-aa3417d6d2c9.png)

2. If I set the elevation > 0, the desired elevation is kept when rotating...
![image](https://user-images.githubusercontent.com/596903/59526882-5913df00-8eda-11e9-8bb8-dca9a193ecab.png)


This is the expecting behavior. I don't really understand what is your issue.
